runtime/doc/treesitter.txt

```diff
@@ -475,10 +497,10 @@ The following captures are linked by default to standard |group-name|s (use
 @comment                line and block comments
 @comment.documentation  comments documenting code
 
-@comment.error          error-type comments (e.g. `ERROR`, `FIXME`, `DEPRECATED:`)
+@comment.error          error-type comments (e.g. `ERROR`, `FIXME`)
```

Comment:
Please keep that, it's added deliberately. (used in `vimdoc`).

Reply:
Sorry, my bad I missed it's being used in vimdoc. I also removed all the colon in this block for consistency

Reply:
Well, the colons _are_ captured ;) But I agree, that's not relevant here and looks better this way.